go/vt/vttablet/onlineddl/executor.go

```diff
@@ -3518,6 +3532,10 @@ func (e *Executor) reviewRunningMigrations(ctx context.Context) (countRunnning i
 	if err != nil {
 		return countRunnning, cancellable, err
 	}
+	pendingMigrationsUUIDs, err := e.readPendingMigrationsUUIDs(ctx)
```

Comment:
Same note here about enforcing/guaranteeing submission order by adding an ORDER BY clause to the query that gets used.

Reply:
You must have missed it, all relevant queries have `ORDER BY id`